README.md

```diff
@@ -34,14 +34,14 @@ If you get any error logging in, please email `consult@nersc.gov` with details a
 
 Time | Event
 -----|--------
-13:00 | Arrive and Set Up
-13:30 | Meeting Overview
-13:45 | Introduction to TOAST
-14:45 | Simulated Scan Strategies
-15:45 | Coffee Break
-16:00 | Simulated Sky Signals
-17:00 | Hackathon Ideas and Discussion
-17:30 | End of Day 1
+13:30 | Arrive and Set Up
+14:00 | Meeting Overview
+14:15 | Introduction to TOAST
+15:15 | Simulated Scan Strategies
+16:15 | Coffee Break
+16:45 | Simulated Sky Signals
+17:45 | Hackathon Ideas and Discussion
+18:45 | End of Day 1
```

Comment:
shorten coffee breaks, finish by 18:15?

Reply:
Ok, fine for me, when I changed all of them I was aiming for consistency.